source/includes/api-reference/_api-reference.html.md

```diff
@@ -229,6 +254,153 @@ This message has no extra data.
 #### TRAINING_INITIALIZED
 This message has no extra data.
 
+
+# BRAIN Metrics
+
+The metrics endpoints will return data from the specified version of a BRAIN's
+training. They can be useful for analyzing what happened during training.
+
+## Episode Value
+
+Episode value contains data about each training episode for a given version of a BRAIN.
+This represents what the AI has been rewarded and what it's currently learning
+through simulation.
+
+> Request
+
+```text
+GET /v1/{userName}/{brainName}/{version}/metrics/episode_value
+```
+
+| Parameter | Description |
+| --- | --- |
+| userName | Name of the user who has the BRAIN |
+| brainName | Name of the BRAIN |
+| version | Version of the BRAIN |
+
+> Example JSON Response
+
+```json
+[
+  {
+    "episode": 1,
+    "lesson": "balancing",
+    "value": 25,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.712068096Z"
+  },
+  {
+    "episode": 2,
+    "lesson": "balancing",
+    "value": 43,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.739321856Z"
+  },
+  {
+    "episode": 3,
+    "lesson": "balancing",
+    "value": 16,
+    "concept": "balance",
+    "time": "2017-11-10T06:37:11.769031936Z"
+  }
+]
+```
+
+### Response
+
+| Parameter | Description |
+| --- | --- |
+| value | The episode's cumulative reward value |
+| episode | The episode for this value |
+| iteration | Total number of simulator iterations that have occurred |
+| time | Timestamp for when this value occurred |
+| concept| The concept this value corresponds to |
+| lesson | The lesson this value corresponds to |
+
+## Test Pass Value
+
+Test pass value contains data for test passes that occur once every 20 episodes
+during training for a given version of a BRAIN.
+This represents the AI's highest reward value achieved at a given time.
+
+> Request
+
+```text
+GET /v1/{userName}/{brainName}/{version}/metrics/test_pass_value
+```
+
+| Parameter | Description |
+| --- | --- |
+| userName | Name of the user who has the BRAIN |
+| brainName | Name of the BRAIN |
+| version | Version of the BRAIN |
+
+> Example JSON Response
+
+```json
+   Example needed
+```
+
+### Response
+
+| Parameter | Description |
+| --- | --- |
+| value | The episode's cumulative reward value |
+| episode | The test pass episode for this value |
+| iteration | Total number of simulator iterations that have occurred |
+| time | Timestamp for when this value occurred |
+| concept| The concept this value corresponds to |
+| lesson | The lesson this value corresponds to |
+
+## Iterations
+
+Iterations contains data for the number of iterations that have occurred in a
+simulation at a given time period.
```

Comment:
What is this based on? A certain time period? For an episode? What are they broken up by?

Reply:
This is where I'm not certain: I thought it used to mark every 100 or 1000 iterations or something, but I think it shifted to be like episode end now, which doesn't seem super useful because you can get that as part of the other metrics now.. Because I'm not sure I don't really know what to edit here/I don't want to be wrong